### PR TITLE
Add create bounty modal

### PR DESCRIPTION
**Pull request by MiZiet**
Add new modal to propose bounties from app. Validation - to be added soon.

### CODE REVIEW

File: packages/apps/public/locales/en/app-bounties.json [hunk 1]
```diff
@@ -1,11 +1,22 @@
 {
+  "Add Bounty": "Add Bounty",
+  "Add bounty": "Add bounty",
```
Comment (krzysztof-jelski) [inline packages/apps/public/locales/en/app-bounties.json:3]: the one with capital 'B' should be enough

File: packages/page-bounties/src/BountyCreate.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  const bountyDepositPerByte = api.consts.treasury.dataDepositPerByte;
+
+  const [accountId, setAccountId] = useState<string | null>(null);
+  const [title, setTitle] = useState('');
+  const [bond, setBond] = useState(bountyDepositBase.toBn());
+  const [amount, setAmount] = useState<BN | undefined>(BN_ZERO);
+  const [isValid] = useState(true);
+  const [isOpen, toggleIsOpen] = useToggle();
+
+  const onTitleChange = (value: string) => {
```
Comment (jacogr) [inline packages/page-bounties/src/BountyCreate.tsx:27]: useCallback rather.

Comment (jacogr) [reply packages/page-bounties/src/BountyCreate.tsx:27]: ... which would probably meant you don't want to pass the consts `bountyDepositBase`, `bountyDepositPerByte` in, but rather the API. The former will change on each render, the later won't.

**Comment by jacogr** [comment]
Happy as-is. If you want it in (with some work left), happy to get it in as-is since it does move things forward.

**Comment by krzysztof-jelski** [comment]
Great, let's merge it in, then. We'll add validation in another PR.

**Review by jacogr — APPROVED** [review]
Excellent, nice move forward.

**Pull request merged**